cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -175,6 +178,12 @@
 
     private static final String NAME_SUFFIX = "-kafka";
 
+    private static final String SECRET_JMX_SECRET_SUFFIX = NAME_SUFFIX + "-jmx";
+    private static final String SECRET_JMX_USERNAME_KEY = "jmx-username";
+    private static final String SECRET_JMX_PASSWORD_KEY = "jmx-password";
+    private static final String ENV_VAR_KAFKA_JMX_USERNAME = "KAFKA_USERNAME";
+    private static final String ENV_VAR_KAFKA_JMX_PASSWORD = "KAFKA_PASSWORD";
```

Comment:
I would use `KAFKA_JMX_USERNAME` and `KAFKA_JMX_PASSWORD` as values

Reply:
Agreed!